README.md

```diff
@@ -110,3 +110,21 @@ Note: This is only useful if you have access to a production running on Render.c
 
 1. Add image in both .png and .avif with the correct weapon ID (`replace-img-names.ts` can help)
 2. Create new weapon ids json and weapon translation jsons using `create-weapon-json.ts` script
+
+### Fix errors from the CI Pipeline
+
+#### Prettier step
+
+If you change any files and the CI pipeline errors out on the `Prettier` step, run this command in the repo's root directory:
+
+```sh
+npm run prettier:write
+```
+
+#### Stylelint step
+
+If you modified any `.css` or `.scss` files and the CI pipeline errors out on the `Stylelint` step, run this command in the repo's root directory:
+
+```sh
+npm run lint:styles-fix
+```
```

Comment:
Should maybe instead point to `npm run cf` which runs all the same checks as CI and fixes them :) But it's true would be nice to document it.

Reply:
Oh, I didn't even know about that command - I will remove my command then & then update the README with that info. Thanks!

Reply:
I found a bit of an issue with running npm run cf - it changed almost every single file cause of me being on Windows & the line-endings being different than the ones stored on Git (LF)

So.....maybe it's best not to ask users to run npm run cf on their end? And instead run the appropriate command on their end instead of doing everything at once

Reply:
Handled in these PRs:
- https://github.com/Sendouc/sendou.ink/pull/1092 
- https://github.com/Sendouc/sendou.ink/pull/1099